### DISCUSSION TITLE
Error to correct for Object.create

### DISCUSSION THREAD

**Discussion by Pierre-Thibault** [mdn/mdn-community#596] [Content] [answered]
I don't know how to edit the first code example at [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create). It says:  "// inherited properties can be overwritten" but there is no property overwritten on this line. It creates a new property at the object level. The property is still present in the prototype and can be access using `Object.getPrototypeOf(me).isHuman`. Maybe it should say that properties can be override and that the original property can still be accessed in the prototype.

**Comment by AnilSeervi** [comment, marked as answer]
At first, we have an Object Literal named `person`,
```js
const person = {
  isHuman: false,
  printIntroduction: function() {
    console.log(`My name is ${this.name}. Am I human? ${this.isHuman}`);
  }
};
```
On **line: 8**, in the demo example, a new object is created from the existing `person` object using the `Object.create()` method.
```js
const me = Object.create(person);
```
> Note that this creates a new object and doesn't just reference the `person` object.

On **line: 10**, you see a new property is being set on the `me` object, but not on `person`
```js
me.name = 'Matthew'; // "name" is a property set on "me", but not on "person"
```

The same goes with overwriting the already defined properties, On **line: 11**
```js
me.isHuman = true; // inherited properties can be overwritten
```

And on **line: 13**, the `printIntroduction()` method is invoked, which just logs the _expected output:_
```js
me.printIntroduction();
// expected output: "My name is Matthew. Am I human? true"
```

---

_Hope this helps you understand what the example is trying to convey!_

**Reply by Pierre-Thibault** [reply]
No, at line 11, the inherited property is not overwritten. We still can access it:

    Object.getPrototypeOf(me).isHuman;

It overrides the property.

**Reply by AnilSeervi** [reply]
`Object.getPrototypeOf()` will return you the prototype(object) from which it was created(`Object.create()`)

So, According to the [demo example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create), `Object.getPrototypeOf(me)` will return you the `person` object. You can check by the following:
```js
Object.getPrototypeOf(me) === person    // true
```
and in your previous comment you are checking `isHuman` property on the `person` object, which is not overwritten.

**Reply by Pierre-Thibault** [reply]
Yes, it is the point. Nothing is overwritten. `me.isHuman = true` overrides the property from the prototype. The document says "overwritten" and it should say overridden. At least it is what I believe to be appropriate.

**Reply by AnilSeervi** [reply]
IDK, **overwritten** seems fine to me.
StackOverflow - https://stackoverflow.com/questions/8651562/overwrite-or-override

**Reply by Pierre-Thibault** [reply]
OK. I disagree. Since the previous value is still available. I think override would be more appropriate.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
